Makefile

```
@@ -0,0 +1,24 @@
+PROJECT_NAME=$(shell basename "$(PWD)")
+
+tidy:
+	go mod tidy
+
+vet:
+	go vet ./...
+
+test:
+	go test ./... -v
+
+BINARIES_DIRECTORY=bin
+MAIN_FILE=cmd/main.go
+CONFIG_FILE=examples/minimal.toml
```

```suggestion
CONFIG_FILE ?= examples/minimal.toml
```
Then it can be runned as `CONFIG_FILE=dev.toml make run`